### DISCUSSION TITLE
Add custom tag sentences

### DISCUSSION THREAD

**Discussion by IkanTerbakor** [Vdauphin/HeartsAndMinds#1670] [Question and answer]
Hello!

How do i add custom spray tag sentences?

Cheers!

**Comment by Vdauphin** [comment]
Hello,

You need to add a new string here: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L332C5-L342
And define in the stringtable the sentence like here: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/stringtable.xml#L2525-L2528

Cheers
